packages/mediapipe-core/lib/src/task_options.dart

```diff
@@ -0,0 +1,114 @@
+import 'dart:typed_data';
+import 'package:equatable/equatable.dart';
+import 'mediapipe_common_bindings.dart' as bindings;
+import 'ffi_utils.dart';
+
+class BaseOptions extends Equatable {
+  const BaseOptions({this.modelAssetBuffer, this.modelAssetPath})
+      : assert(
+          !(modelAssetBuffer == null && modelAssetPath == null),
```

Comment:
Nit for readability:
```suggestion
          modelAssetBuffer != null || modelAssetPath != null,
```

Reply:
I tripped over both of these asserts trying to unpack them, double checking them... this ensures at least one is provided